UPGRADE.asciidoc

```diff
@@ -68,6 +68,13 @@ Upgrading from a version prior to X.Y.Z
 
 Once completed, update the file /usr/local/pf/conf/currently-at to match the new release number (PacketFence X.Y.Z).
 
+Samba cache directory changed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Rejoining the domains from PacketFence GUI is required. 
```

Comment:
They should also run the 'Refresh domain configuration' as well on the same page